Doc/whatsnew/3.8.rst

```diff
@@ -1456,6 +1603,11 @@ Deprecated
   constant nodes.
   (Contributed by Serhiy Storchaka in :issue:`36917`.)
 
+* The :func:`asyncio.coroutine` :term:`decorator` is deprecated and will be
+  removed in version 3.10.  Instead of ``@asyncio.coroutine``, use
+  :keyword:`async def` instead.
+  (Contributed by Andrew Svetlov in :issue:`36921`.)
+
 * The following functions and methods are deprecated in the :mod:`gettext`
   module: :func:`~gettext.lgettext`, :func:`~gettext.ldgettext`,
   :func:`~gettext.lngettext` and :func:`~gettext.ldngettext`.
```

Comment:
I think the link shortening can probably be removed here, as it would not be excessively long.

Reply:
Seems fine to me.  Please don't debate every nit.  It just eats time.

Reply:
@rhettinger 
> Please don't debate every nit. It just eats time.

I apologize if this came across as nitpicking. My only motivation was to understand some of the reasons behind the formatting details. I'll leave you to finish it, thanks for  including some of my proposed changes in https://github.com/python/cpython/pull/16535.